cluster/manifests/skipper/deployment.yaml

```diff
@@ -211,7 +211,4 @@ spec:
       - effect: NoSchedule
         key: dedicated
         value: skipper-ingress
-{{ else }}
-      nodeSelector:
-        node.kubernetes.io/role: worker
```

Comment:
This was dropped because it's not really needed in the general case and I just discovered we had it because it caused issues for the spot.io integration.